### ISSUE TITLE
methods for betadisper objects fail when bias.adjust=TRUE

### ISSUE THREAD

**Issue by jrcunning** [vegandevs/vegan#219]
Using the sample code from the permutest.betadisper help text, if the bias.adjust=TRUE is included when building the betadisper object "mod", downstream anova and permutest methods fail, giving this error:

`Error in eval(expr, envir, enclos) : object 'Distances' not found`

```R
data(varespec)

## Bray-Curtis distances between samples
dis <- vegdist(varespec)

## First 16 sites grazed, remaining 8 sites ungrazed
groups <- factor(c(rep(1,16), rep(2,8)), labels = c("grazed","ungrazed"))

## Calculate multivariate dispersions
mod <- betadisper(dis, groups, bias.adjust=TRUE)
mod

## Perform test
anova(mod)

## Permutation test for F
pmod <- permutest(mod, permutations = 99, pairwise = TRUE)
```

**Comment by jarioksa** [comment]
This is a subtle issue: **R** objects are not always what they look like. We needed counts of observations for groups, and we used `table()` for that purpose. Although the output of `table` looks like an ordinary vector, it is a vector of class `"table"` and has a specific handling in `data.frame`. When we then built an internal `data.frame` for `anova.betadisper()` or `permutest.betadisper` the result was not like we expected.

I have a fix for this (commit a0e16ea), but we got to see that this fix does not break anything else before merging this.

Thanks for reporting this problem. We are going to have this fix in **vegan** release 2.4-2 before Christmas (or what ever you call it). 

(cc @adrianstier @bbolker)
